### ISSUE TITLE
Bug with Quickstart / Interactive Animation: java.lang.RuntimeException: Native exception: Unable to get platform id in /home/bschmid/3D_Animation/3D_Animation_Native/Raycaster.cpp (line 73)

### ISSUE THREAD

**Issue by stevenjwest** [bene51/3Dscript#21]
Hi,

I have just installed your great-looking package into ImageJ.  I have tried to follow the Quickstart example with T1 Head data, as specified on the ImageJ website for this package:

https://imagej.net/3Dscript

However, when I run `Interactive Animation`, I get the following exception:

(Fiji Is Just) ImageJ 2.0.0-rc-69/1.52p; Java 1.8.0_172 [64-bit]; Linux 4.15.0-66-generic; 300MB of 39417MB (<1%)
 
java.lang.RuntimeException: Native exception: Unable to get platform id in /home/bschmid/3D_Animation/3D_Animation_Native/Raycaster.cpp (line 73)
	at animation3d.renderer3d.OpenCLRaycaster.initRaycaster16(Native Method)
	at animation3d.renderer3d.OpenCLRaycaster.<init>(OpenCLRaycaster.java:85)
	at animation3d.renderer3d.Renderer3D.<init>(Renderer3D.java:27)
	at animation3d.main.InteractiveRaycaster.run(InteractiveRaycaster.java:151)
	at ij.plugin.filter.PlugInFilterRunner.processOneImage(PlugInFilterRunner.java:266)
	at ij.plugin.filter.PlugInFilterRunner.<init>(PlugInFilterRunner.java:114)
	at ij.IJ.runUserPlugIn(IJ.java:232)
	at ij.IJ.runPlugIn(IJ.java:193)
	at ij.Executer.runCommand(Executer.java:137)
	at ij.Executer.run(Executer.java:66)
	at java.lang.Thread.run(Thread.java:748)

The exception provides my runtime environment too.

It seems that the animation3d package has a hard-coded path in it, which obv will not exist on my machine.

Cheers,

Steve J. West

**Comment by bene51** [comment]
Hi Steve,
my first guess is that your graphics driver is not up-to-date. Which graphics card do you have? The path you are seeing is not hard-coded but just used to generate the error message, so it has nothing to do with it.
Cheers

**Comment by stevenjwest** [comment]
Hi Ben,

Great thanks for your quick reply.  I have a NVIDIA Corporation GP108 [GeForce GT 1030] graphics card, but I checked the install and apparently the driver is not installed on my computer.

Just for any future reference, I followed the instructions on this webpage (https://www.linuxbabe.com/ubuntu/install-nvidia-driver-ubuntu-18-04) to install the NVIDIA driver for my system.

The plugin now works!  Thanks!
